### DISCUSSION TITLE
Is the source code available for review?

### DISCUSSION THREAD

**Discussion by mindware** [ttscoff/bunch-docs#263] [Q&A] [answered]
I'm considering using Bunch. Is the source code available for review?

**Comment by ttscoff** [comment, marked as answer]
Nope. That’s a strange requirement before using an app…

Thanks,
Brett

**Reply by mindware** [reply]
Thanks for clarifying. Good to know.
I don’t think it’s a weird request, but I was surprised it hadn’t come up before on these discussions. I prefer to use products that have their source code available for public scrutiny, personally.
Thanks.
